docs/developer-docs/latest/update-migration-guides/migration-guides/v4/plugin/update-folder-structure.md

```diff
@@ -0,0 +1,356 @@
+---
+title: v4 Plugin Migration - Update folder structure - Strapi Developer Docs
+description:
+canonicalUrl:
+sidebarDepth: 2
+---
+
+<!-- TODO: update SEO -->
+
+# v4 plugin migration: Update the folder structure
```

Comment:
Maybe it is the correct place to also say that if the plugin is aimed to be on npm, then the `package.json` file should have the `strapi.kind` key set to `"plugin"`: `{ "strapi": { "kind": "plugin" } }`

Reply:
Thanks, added it to enabling a plugin as it's more related to this section and the auto-discovery feature.